crates/leader-elector/src/main.rs

```diff
@@ -9,11 +9,15 @@ use serde::Deserialize;
 use std::{fs, time::Duration};
 use tracing::{debug, info, warn};
 
-#[derive(Debug, Deserialize)]
+#[derive(Debug, Deserialize, Clap)]
```

Comment:
```suggestion
#[derive(Debug, Clap)]
```

We no longer need it be deserializable, that's for Envy.

Reply:
Correct, thanks for pointing it out. Removed.